src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py

```diff
@@ -198,6 +198,17 @@ def test_containerapp_env_dapr_components(self, resource_group):
             JMESPathCheck('length(@)', 0),
         ])
 
+        self.cmd('containerapp env dapr-component init -n {} -g {}'.format(env_name, resource_group), checks=[
```

Comment:
Please rerun this test lively with `--live`:
`azdev test test_containerapp_env_dapr_components --live`
https://github.com/Azure/azure-cli-dev-tools#style-linter-check-and-testing